modules/xfem/src/userobjects/InterfaceMeshCut3DUserObject.C

```diff
@@ -0,0 +1,406 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "InterfaceMeshCut3DUserObject.h"
+#include "XFEMMovingInterfaceVelocityBase.h"
+#include "libmesh/plane.h"
+#include "libmesh/enum_point_locator_type.h"
+#include "XFEMFuncs.h"
```

Comment:
This include can be moved into the header of the base class.

Reply:
done.